src/spas/libs/member-management/feature/src/lib/member-management/user.service.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { TestBed } from '@angular/core/testing';
+
+import { UserService } from './user.service';
+
+describe('UserService', () => {
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  it('should be created', () => {
+    const service: UserService = TestBed.get(UserService);
+    expect(service).toBeTruthy();
+  });
+});
```

Comment:
should be moved to the member-management/domain lib along with the `user.service.ts` service